README.md

```diff
@@ -1,6 +1,6 @@
 # Kompose (Kubernetes + Compose)
 
-[![Build Status Widget]][Build Status] [![Coverage Status Widget]][Coverage Status] [![GoDoc Widget]][GoDoc] [![Slack Widget]][Slack]
+[![Build Status Widget]][Build Status] [![Coverage Status Widget]][Coverage Status] [![GoDoc Widget]][GoDoc] [![Slack Widget]][Slack]  [![GoReportCard Widget]][GoReportCardResult]
```

Comment:
One last nitpick (sorry!), mind moving slack to being the last widget / move goreportcard forward? Other than that, LGTM after this has been changed!